.internal-ci/charts/reserve-auditor/values.yaml

```diff
@@ -45,7 +45,7 @@ ledgerScanner:
   image:
     repository: mobilecoin/reserve-auditor
     pullPolicy: Always
-    tag: v0.0.0-test
+    tag: ""
```

Comment:
@aweis89 what does an empty tag signify here, and does removing the line have the same impact?

Reply:
Conventionally this indicates the expected value to be used. It's set to an empty string by default because this tag is set dynamically to the chart version which is the new release